includes/class-wc-payments-features.php

```diff
@@ -87,6 +89,27 @@ public static function is_account_overview_task_list_enabled() {
 		return get_option( '_wcpay_feature_account_overview_task_list' );
 	}
 
+	/**
+	 * Checks whether WCPay Subscriptions is enabled.
+	 *
+	 * @param WC_Payments_Account $wcpay_account Connected WCPay Account to check the location.
+	 * @return bool
+	 */
+	public static function is_wcpay_subscriptions_enabled( $wcpay_account = null ) {
+		$is_wcpay_subscriptions_enabled = get_option( self::WCPAY_SUBSCRIPTIONS_FLAG_NAME );
```

Comment:
**Nitpick:** Looks like the convention in this file is to set `'0'` as the default here and then perform a strict comparison against `'1'` in the return statement.